js/sow.google-map.js

```diff
@@ -378,9 +380,16 @@ sowb.SiteOriginGoogleMap = function($) {
 						locationPromise.resolve( results[ 0 ].geometry.location );
 					} else if ( status === google.maps.GeocoderStatus.OVER_QUERY_LIMIT ) {
 						//try again please
-						setTimeout( function () {
-							this.getGeocoder().geocode.call( this, location, onGeocodeResults );
-						}.bind( this ), 100 );
+						if ( geocodeTimer == null ) {
+							geocodeTimer = setInterval( function () {
+								if ( geocodeIteration <= 2 ) {
+									geocodeIteration++;
+									this.getGeocoder().geocode.call( this, location, onGeocodeResults );
+								} else {
+									clearInterval( geocodeTimer );
+								}
+							}.bind( this ), 100 );
+						}
 					} else if ( status === google.maps.GeocoderStatus.ZERO_RESULTS ) {
 						locationPromise.reject( status );
```

Comment:
@braamgenis Does this stop processing the rest of markers? If so, maybe it's best to do that as if the API is over the query limit and nothing can process there's really no point in proceeding.

Reply:
Sorry, I forgot to reply to this comment. This is just for the current marker, so requests for other markers will still occur. That's a good suggestion, but it will require a larger refactor to stop all other requests if we hit the `OVER_QUERY_LIMIT` status, which should perhaps be part of the work to move geocoding requests to the server-side. I'll try look into this soon.